earth_enterprise/src/installer/install_fusion.sh

```diff
@@ -177,17 +177,17 @@ main_install()
 	# add group if it does not exist
 	if [ -z "$GROUP_EXISTS" ]; then
 		groupadd -r $GROUPNAME &> /dev/null 
-        NEW_GEGROUP=true 
+		NEW_GEGROUP=true 
 	fi
 
 	# add user if it does not exist
 	if [ -z "$USERNAME_EXISTS" ]; then
-        mkdir -p $BASEINSTALLDIR_OPT/.users/$GEFUSIONUSER_NAME
-		useradd -d $BASEINSTALLDIR_OPT/.users/$GEFUSIONUSER_NAME -g gegroup $GEFUSIONUSER_NAME
-        NEW_GEFUSIONUSER=true
-    else
-        # user already exists -- update primary group
-        usermod -g $GROUPNAME $GEFUSIONUSER_NAME
+		mkdir -p $BASEINSTALLDIR_OPT/.users/$GEFUSIONUSER_NAME
+		useradd --home $BASEINSTALLDIR_OPT/.users/$GEFUSIONUSER_NAME --system --gid gegroup $GEFUSIONUSER_NAME
```

Comment:
@asmitaw The groupname is a different issue, so created a tracking ticket #150. I'll look to add that, but it complicates the testing of this change a bit.

I don't know about the user directories - they don't seem to be used, but this is how it was from the initial commit of that installer script: https://github.com/google/earthenterprise/blob/6b20145854c41daa25cd365698554594ab5941d0/earth_enterprise/src/installer/install_fusion.sh#L203 
Perhaps @arjntc1  or @tst-ccamp  can advise?